src/main/java/com/wavefront/sdk/common/clients/service/ReportingService.java

```diff
@@ -128,16 +130,18 @@ private int safeGetResponseCodeAndClose(HttpURLConnection urlConn) {
     try {
       statusCode = urlConn.getResponseCode();
     } catch (IOException ex) {
-      log.log(Level.SEVERE, "Unable to obtain status code from the Wavefront service at "
-          + urlConn.getURL().toString(), ex);
+      MESSAGE_SUPPRESSING_LOGGER.log(urlConn.getURL().toString(), Level.SEVERE,
+          "Unable to obtain status code from the Wavefront service at "
+              + urlConn.getURL().toString());
```

Comment:
Let's log the exception message to provide some additional context.  E.g., ` + " due to: " + ex.getMessage()`